updating the test cases (#7448)
desktop/desktop: docs/process/testing.md

      - [ ] `Cancel` button does not save any changes made; modal closed
    - [ ] Appearance
      - [ ] Light theme is default
      - [ ] Dark theme is optional 

>       - [ ] For MAC OS, users can opt to match system prefence theme with checkbox

Reviewer: ```suggestion
      - [ ] For Mac, users can opt to match system preference theme with checkbox
```

    - [ ] Merge hint shows status and branches to be merged
    - [ ] `Merge` button only activated if something to merge (includes awaiting conflicts)
      - [ ] If merged, success banner is shown temporarily; user can `X` to remove

>       - [ ] If conflicts, conflict modal shows quanity of files, ability to open in [editor], open in command line, `Abort` button, while `Commit merge` button is deactivated until files are resolved; Changes tab shows all files 

Reviewer 2: ```suggestion
      - [ ] If conflicts, conflict modal shows quantity of files, ability to open in [editor], open in command line, `Abort` button, while `Commit merge` button is deactivated until files are resolved; Changes tab shows all files 
```

        - [ ] If conflict resolved, files marked green, and user can click `Commit merge` button
	    - [ ] If merged, success banner is shown temporarily; user can `X` to remove
	    - [ ] Binary files must be resolved in command line before committing merge
        - [ ] Aborting partially resolved commit surfaces "Are you sure?" dialogue; `Cancel` or `Abort merge` buttons
  - [ ] Rebase Current Branch...

>     - [ ] Use can filter to find existing branches

Reviewer 3: ```suggestion
    - [ ] User can filter to find existing branches
```

    - [ ] User can select branch, other than current one
    - [ ] Rebase hint shows status and branches to be merged
    - [ ] `Start rebase` button only activated if something to rebase
      - [ ] If rebase, success banner is shown temporarily; user can `X` to remove

>       - [ ] If conflicts, conflict modal shows quanity of files, ability to open in [editor], open in command line, `Abort` button, while `Continue rebase` button is deactivated until files are resolved

Reviewer 3: ```suggestion
      - [ ] If conflicts, conflict modal shows quantity of files, ability to open in [editor], open in command line, `Abort` button, while `Continue rebase` button is deactivated until files are resolved
```

>         - [ ] If rebase resolved, files marked green, and user can click `Continue rebase` button

Reviewer 3: ```suggestion
        - [ ] If conflicts resolved, files marked green, and user can click `Continue rebase` button
```

	    - [ ] If rebased, success banner is shown temporarily; user can `X` to remove

> 	    - [ ] Binary files must be resolved in command line before committing rebase

Reviewer 3: Is this accurate? Don't we have support for choosing binary files in desktop now?

Author: You are correct. I had copy/pasted the merge conflict steps for rebasing. Fixing it.

  - [ ] `Report Issue...` opens issue filing in Desktop repository on `github.com`
  - [ ] `Contact GitHub Support...` opens `https://github.com/contact` page with user and build prepopulated
  - [ ] `Show User Guides` opens Desktop help page on `github.com`

>   - [ ] `Show Keyboard Shurtcuts` opens `https://help.github.com/en/desktop/getting-started-with-github-desktop/keyboard-shortcuts-in-github-desktop`

Reviewer 3: ```suggestion
  - [ ] `Show Keyboard Shortcuts` opens `https://help.github.com/en/desktop/getting-started-with-github-desktop/keyboard-shortcuts-in-github-desktop`
```

  - [ ] `Show Logs in Finder/Explorer` opens Finder/Explorer logs in local directory
    - [ ] Mac: `ls ~/Library/Application\ Support/GitHub\ Desktop/Logs/*.log`
    - [ ] Windows: `%LOCALAPPDATA%\\Desktop\\*.desktop.production.log`
  - [ ] About GitHub Desktop (Windows only)
  
### Next Steps (wip)

### Repositories list
  - [ ] Current repository is always shown in top slot with respective icon; if repository exists

>   - [ ] Opening list shows all repositories in categorized by owner in alpha format with a working filter

Reviewer 3: ```suggestion
  - [ ] Opening list shows all repositories, categorized by owner in alpha format with a working filter
```

	- [ ] User can open in finder, preferred editor, or OS default program  
  - [ ] Panes can be resized horizontally, and contents resize to take the full width
    - [ ] Quitting Desktop and relaunching remembers pane sizes

>   - [ ] Uncommited files are stashing if user attempts to switch branches

Reviewer 3: ```suggestion
  - [ ] Uncommitted files are optionally stashed if user attempts to switch branches
```

    - [ ] Modal asks user to stash on current branch or bring changes to new branch; `Cancel` or `Switch Branch` buttons

>       - [ ] If stashed then changes shown under Stashed Changes section of the under Changes tab

Reviewer 3: ```suggestion
      - [ ] If stashed then changes shown under Stashed Changes section below Changes tab when returning to the original branch
```

>         - [ ] Stashed changes section show all stashed files; user can discard or restore to Changes

Reviewer 3: ```suggestion
        - [ ] Stashed changes section show all stashed files and diffs; user can discard or restore to Changes
```

>       - [ ] If moved to new branch, files under Changes tab

Reviewer 3: ```suggestion
      - [ ] If moved to new branch, all previously changed files will exist under Changes tab on the new branch
```
